presale/models/presale_order.py

```diff
@@ -0,0 +1,89 @@
+from odoo import api, fields, models
+
+
+class PresaleOrder(models.Model):
+    _name = 'presale.order'
+    _description = 'presales orders'
+
+    name = fields.Char(string='Preorder Reference',
+                       required=True, copy=False, readonly=True,
+                       default=lambda self: 'New')
+
+    active = fields.Boolean('Active', default=True)
+
+    state = fields.Selection(
+        selection=[
+            ('draft', 'Draft'),
+            ('confirmed', 'Confirmed')
+        ],
+        string='Status',
+        readonly=True, copy=False,
+        default='draft'
+    )
+
+    partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        string="Customer",
+        required=True, readonly=False, change_default=True)
+
+    order_lines = fields.One2many(
+        'presale.order.line', 'order_id', string='Order Lines')
+
+    sale_order_id = fields.Many2one(
+        'sale.order', string='Sale Order', copy=False)
+
+    def validate_action(self):
+        return True
```

Comment:
This is overwritten by a method with the same name on line 47, this one can be removed entirely (and it does nothing anyway)